lib/utils/prompt/confirmation_test.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package prompt
+
+import (
+	"context"
+	"io"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestInput(t *testing.T) {
+	t.Parallel()
```

Comment:
I also think for this to work as probably intended the `t.Parallel()` call should be put inside all the individual `t.Run` test functions.

Reply:
The tests here are intentionally sequential, because they reuse the same `io.Pipe` and `prompt.ContextReader`.
This whole test will run in parallel with other top-level tests in this package as is though.